daxgen/daxgen.py

```diff
@@ -164,6 +161,20 @@ def _check_sanity(self):
             except nx.NetworkXError:
                 raise ValueError("Graph is not bipartite.")
 
+            U, V = nx.bipartite.sets(self.graph)
+
+            # Select an arbitrary vertex from the set U and based on its
+            # attributes decide if the set U represents files and respectively,
+            # V represents tasks, or the other way round.
+            v = next(iter(U))
+            node_attrs = set(self.graph.node[v].keys())
+            files, tasks = (U, V) if node_attrs >= _FILE_ATTRS else (V, U)
```

Comment:
yes, maybe move it to the method itself especially if meant to be private and not a constant through the code, unless you want to add other attributes later on. Or you can use it every time you check for this particular attribute in other parts of the code